requirements-dev.txt

```diff
@@ -1,6 +1,7 @@
 cvxpy>=1.0.15
-cvxopt>=1.2.3
+numpy>=1.13
```

Comment:
This and scipy below are redundant, the requirements are already listed in the package itself: https://github.com/Qiskit/qiskit-ignis/blob/master/setup.py#L22 we don't need to specify it in the development/testing requirements too.

Reply:
I'm trying to force conda to install MKL in the azure tests to see if they pass